### ISSUE TITLE
Claims showing outdated/incorrect information

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/chainquery#75]
I bet there are more of these instances. The claim below is showing a fee of 1 and publisher that's not on the current claim. 

https://chainquery.lbry.io/api/sql?query=SELECT%20*%20FROM%20claim%20where%20claim_id=%22008401a48cecef2a13cf5376b696169bb36b18c6%22

```{
  "success": true,
  "error": null,
  "data": [
    {
      "author": "",
      "bid_state": "Controlling",
      "certificate": null,
      "claim_address": "bLa8yDqD4FCxxmL7j3pfvXBKghwvDEanMZ",
      "claim_id": "008401a48cecef2a13cf5376b696169bb36b18c6",
      "claim_type": 1,
      "content_type": "video/mp4",
      "created_at": "2018-04-20T17:03:02Z",
      "description": "Thanks for all the tips for the Holly video. She bought a Nintendo Switch with the money at xmas.\n\nThanks again",
      "effective_amount": 11008740000001,
      "fee": "1.00000000",
      "fee_address": "bVkcRcSxB6UQpw5wbv8Sd7C4ezsMevGEmg",
      "fee_currency": "USD",
      "height": 477596,
      "id": 1090,
      "is_cert_processed": 1,
      "is_cert_valid": 0,
      "is_filtered": 0,
      "is_nsfw": 0,
      "language": "en",
      "modified_at": "2018-11-29T19:29:40Z",
      "name": "one",
      "publisher_id": "71b3ce7e10fa0e25074f6003833224e3b402d76c",
      "publisher_sig": "68b0242f2e3dc5fbfe4219aa6513bb7fa614e46855974eac4796210af9c9121f51ea4c9b58433e326104f4f7f126ae423a6795f42549d2887f06237f81a8ae89",
      "sd_hash": "6a611e6f8ffb4920bf0679de6dee8bf9e56a473f43d8340233934eaf3b7e1211ee5a53c28e33693ca236c1408023d6cd",
      "thumbnail_url": "https://spee.ch/c/holly-happy-working-song",
      "title": "Holly singing The Happy Working Song",
      "transaction_hash_id": "8016efce97d899a2e56d1baee256e6266404cff8596321002af5aa07f740d9f9",
      "transaction_time": 1543483897,
      "valid_at_height": 477596,
      "value_as_hex": "080110011aad02080112e501080410011a24486f6c6c792073696e67696e672054686520486170707920576f726b696e6720536f6e67226f5468616e6b7320666f7220616c6c20746865207469707320666f722074686520486f6c6c7920766964656f2e2053686520626f756768742061204e696e74656e646f20537769746368207769746820746865206d6f6e657920617420786d61732e0a0a5468616e6b7320616761696e2a003214416c6c207269676874732072657365727665642e38004a2a68747470733a2f2f737065652e63682f632f686f6c6c792d68617070792d776f726b696e672d736f6e6752005a001a41080110011a306a611e6f8ffb4920bf0679de6dee8bf9e56a473f43d8340233934eaf3b7e1211ee5a53c28e33693ca236c1408023d6cd2209766964656f2f6d7034",
      "value_as_json": "{\"Claim\":{\"version\":\"_0_0_1\",\"claimType\":\"streamType\",\"stream\":{\"version\":\"_0_0_1\",\"metadata\":{\"version\":\"_0_1_0\",\"language\":\"en\",\"title\":\"Holly singing The Happy Working Song\",\"description\":\"Thanks for all the tips for the Holly video. She bought a Nintendo Switch with the money at xmas.\\n\\nThanks again\",\"author\":\"\",\"license\":\"All rights reserved.\",\"nsfw\":false,\"thumbnail\":\"https://spee.ch/c/holly-happy-working-song\",\"preview\":\"\",\"licenseUrl\":\"\"},\"source\":{\"version\":\"_0_0_1\",\"sourceType\":\"lbry_sd_hash\",\"source\":\"amEeb4/7SSC/Bnnebe6L+eVqRz9D2DQCM5NOrzt+EhHuWlPCjjNpPKI2wUCAI9bN\",\"contentType\":\"video/mp4\"}}}}",
      "version": "_0_0_1",
      "vout": 0
    }
  ]
}```

**Comment by tzarebczan** [comment]
```{
  "success": true,
  "error": null,
  "data": [
    {
      "author": "",
      "bid_state": "Controlling",
      "certificate": null,
      "claim_address": "bYyXpdVTvu95YZBNzcUjBUg9sMAtgPfKdj",
      "claim_id": "d1b53ed6f4903e9ced4bdcba177652e17e64192f",
      "claim_type": 1,
      "content_type": "image/jpeg",
      "created_at": "2018-12-14T17:25:32Z",
      "description": "",
      "effective_amount": 10000000,
      "fee": "0.00000000",
      "fee_address": "",
      "fee_currency": null,
      "height": 485927,
      "id": 557188,
      "is_cert_processed": 1,
      "is_cert_valid": 1,
      "is_filtered": 0,
      "is_nsfw": 0,
      "language": "en",
      "modified_at": "2018-12-14T17:42:36Z",
      "name": "ten1212",
      "publisher_id": "6ac0f69d94bba3491e238f6d5410dde0a24c4617",
      "publisher_sig": "52d9d2fdbde2a70140d5b8425e812b17455f6068d89153df7824fc1b08637b783b22aa5911ab8f390ef1cc0cdaf7185e0c51eb4eae8fdb1c0616fff31140bc36",
      "sd_hash": "2e985f39fb1e2d97f1658379f265e1e024a6a74260f0d3f028b12722083649652a61550f8c6d5b6f23b9ebb05a391394",
      "thumbnail_url": "https://spee.ch/b/he0GRirDLvNMCl9PCAj52MWY.jpg",
      "title": "test upload",
      "transaction_hash_id": "4818c163b29bea78e8ebce3bad9eecbe73a1479a74eac3cfb7d8d41c29d0c1a4",
      "transaction_time": 1544808678,
      "valid_at_height": 485927,
      "value_as_hex": "080110011a990108011251080410011a0b746573742075706c6f616422002a0032044e6f6e6538004a2e68747470733a2f2f737065652e63682f622f68653047526972444c764e4d436c395043416a35324d57592e6a706752005a001a42080110011a302e985f39fb1e2d97f1658379f265e1e024a6a74260f0d3f028b12722083649652a61550f8c6d5b6f23b9ebb05a391394220a696d6167652f6a7065672a5c080110031a4052d9d2fdbde2a70140d5b8425e812b17455f6068d89153df7824fc1b08637b783b22aa5911ab8f390ef1cc0cdaf7185e0c51eb4eae8fdb1c0616fff31140bc3622146ac0f69d94bba3491e238f6d5410dde0a24c4617",
      "value_as_json": "{\"Claim\":{\"version\":\"_0_0_1\",\"claimType\":\"streamType\",\"stream\":{\"version\":\"_0_0_1\",\"metadata\":{\"version\":\"_0_1_0\",\"language\":\"en\",\"title\":\"test upload\",\"description\":\"\",\"author\":\"\",\"license\":\"None\",\"nsfw\":false,\"thumbnail\":\"https://spee.ch/b/he0GRirDLvNMCl9PCAj52MWY.jpg\",\"preview\":\"\",\"licenseUrl\":\"\"},\"source\":{\"version\":\"_0_0_1\",\"sourceType\":\"lbry_sd_hash\",\"source\":\"LphfOfseLZfxZYN58mXh4CSmp0Jg8NPwKLEnIgg2SWUqYVUPjG1bbyO567BaOROU\",\"contentType\":\"image/jpeg\"}},\"publisherSignature\":{\"version\":\"_0_0_1\",\"signatureType\":\"SECP256k1\",\"signature\":\"UtnS/b3ipwFA1bhCXoErF0VfYGjYkVPfeCT8Gwhje3g7IqpZEauPOQ7xzAza9xheDFHrTq6P2xwGFv/zEUC8Ng==\",\"certificateId\":\"asD2nZS7o0keI49tVBDd4KJMRhc=\"}}}",
      "version": "_0_0_1",
      "vout": 0
    }
  ]
}
```

after changing channel to anonymous, cost to 1lbc  (notice it still has channel): 
```
{
  "success": true,
  "error": null,
  "data": [
    {
      "author": "",
      "bid_state": "Controlling",
      "certificate": null,
      "claim_address": "bTdf4z7YMy8mQ6Bqajy6VcZ4qxP8y1ypSb",
      "claim_id": "d1b53ed6f4903e9ced4bdcba177652e17e64192f",
      "claim_type": 1,
      "content_type": "image/jpeg",
      "created_at": "2018-12-14T17:25:32Z",
      "description": "edit metadata, remove channel, add 1 lbc cost",
      "effective_amount": 10000000,
      "fee": "1.00000000",
      "fee_address": "bNFqyeKHJg3kUSaMTdG6cTESHLPYkCf5aA",
      "fee_currency": "LBC",
      "height": 0,
      "id": 557188,
      "is_cert_processed": 1,
      "is_cert_valid": 0,
      "is_filtered": 0,
      "is_nsfw": 0,
      "language": "en",
      "modified_at": "2018-12-21T16:32:50Z",
      "name": "ten1212",
      "publisher_id": "6ac0f69d94bba3491e238f6d5410dde0a24c4617",
      "publisher_sig": "52d9d2fdbde2a70140d5b8425e812b17455f6068d89153df7824fc1b08637b783b22aa5911ab8f390ef1cc0cdaf7185e0c51eb4eae8fdb1c0616fff31140bc36",
      "sd_hash": "2e985f39fb1e2d97f1658379f265e1e024a6a74260f0d3f028b12722083649652a61550f8c6d5b6f23b9ebb05a391394",
      "thumbnail_url": "https://spee.ch/b/he0GRirDLvNMCl9PCAj52MWY.jpg",
      "title": "test upload",
      "transaction_hash_id": "83da50daa9b0d8e7535ee001d0ecc7cee617119293078ba97faefe157778de07",
      "transaction_time": 0,
      "valid_at_height": 485927,
      "value_as_hex": "080110011aed01080112a401080410011a0b746573742075706c6f6164222d65646974206d657461646174612c2072656d6f7665206368616e6e656c2c206164642031206c626320636f73742a0032044e6f6e6538004224080110011a1955688164e6ef2ca1639f2caf2b1cc8d25d59f9f4a2f78af753250000803f4a2e68747470733a2f2f737065652e63682f622f68653047526972444c764e4d436c395043416a35324d57592e6a706752005a001a42080110011a302e985f39fb1e2d97f1658379f265e1e024a6a74260f0d3f028b12722083649652a61550f8c6d5b6f23b9ebb05a391394220a696d6167652f6a706567",
      "value_as_json": "{\"Claim\":{\"version\":\"_0_0_1\",\"claimType\":\"streamType\",\"stream\":{\"version\":\"_0_0_1\",\"metadata\":{\"version\":\"_0_1_0\",\"language\":\"en\",\"title\":\"test upload\",\"description\":\"edit metadata, remove channel, add 1 lbc cost\",\"author\":\"\",\"license\":\"None\",\"nsfw\":false,\"fee\":{\"version\":\"_0_0_1\",\"currency\":\"LBC\",\"address\":\"VWiBZObvLKFjnyyvKxzI0l1Z+fSi94r3Uw==\",\"amount\":1},\"thumbnail\":\"https://spee.ch/b/he0GRirDLvNMCl9PCAj52MWY.jpg\",\"preview\":\"\",\"licenseUrl\":\"\"},\"source\":{\"version\":\"_0_0_1\",\"sourceType\":\"lbry_sd_hash\",\"source\":\"LphfOfseLZfxZYN58mXh4CSmp0Jg8NPwKLEnIgg2SWUqYVUPjG1bbyO567BaOROU\",\"contentType\":\"image/jpeg\"}}}}",
      "version": "_0_0_1",
      "vout": 0
    }
  ]
}
```

after changing channel to a new one, removing cost (fee stayed, channel id updated)
```{
  "success": true,
  "error": null,
  "data": [
    {
      "author": "",
      "bid_state": "Controlling",
      "certificate": null,
      "claim_address": "bZgw77L8C2a14bMmux7FjgqhB7Y3fERamF",
      "claim_id": "d1b53ed6f4903e9ced4bdcba177652e17e64192f",
      "claim_type": 1,
      "content_type": "image/jpeg",
      "created_at": "2018-12-14T17:25:32Z",
      "description": "edit metadata, remove channel, add 1 lbc cost",
      "effective_amount": 10000000,
      "fee": "1.00000000",
      "fee_address": "bNFqyeKHJg3kUSaMTdG6cTESHLPYkCf5aA",
      "fee_currency": "LBC",
      "height": 0,
      "id": 557188,
      "is_cert_processed": 1,
      "is_cert_valid": 1,
      "is_filtered": 0,
      "is_nsfw": 0,
      "language": "en",
      "modified_at": "2018-12-21T16:35:50Z",
      "name": "ten1212",
      "publisher_id": "461b322c4347abf0daaa6c30a384ec791f0b773d",
      "publisher_sig": "8a472ddd917c638592f018c7cf2368650ba1d46468613bd7160c58288c20f717de5e8de7bcf110f09b7a396d54d7043521dcedc526ebc29fa51d7e56e8f0c137",
      "sd_hash": "2e985f39fb1e2d97f1658379f265e1e024a6a74260f0d3f028b12722083649652a61550f8c6d5b6f23b9ebb05a391394",
      "thumbnail_url": "https://spee.ch/b/he0GRirDLvNMCl9PCAj52MWY.jpg",
      "title": "test upload",
      "transaction_hash_id": "0d5267a65a9d10378aa79fa71642fa95b704a6de231e4ca038dbfefc98c10125",
      "transaction_time": 0,
      "valid_at_height": 485927,
      "value_as_hex": "080110011ac6010801127e080410011a0b746573742075706c6f6164222d65646974206d657461646174612c2072656d6f7665206368616e6e656c2c206164642031206c626320636f73742a0032044e6f6e6538004a2e68747470733a2f2f737065652e63682f622f68653047526972444c764e4d436c395043416a35324d57592e6a706752005a001a42080110011a302e985f39fb1e2d97f1658379f265e1e024a6a74260f0d3f028b12722083649652a61550f8c6d5b6f23b9ebb05a391394220a696d6167652f6a7065672a5c080110031a408a472ddd917c638592f018c7cf2368650ba1d46468613bd7160c58288c20f717de5e8de7bcf110f09b7a396d54d7043521dcedc526ebc29fa51d7e56e8f0c1372214461b322c4347abf0daaa6c30a384ec791f0b773d",
      "value_as_json": "{\"Claim\":{\"version\":\"_0_0_1\",\"claimType\":\"streamType\",\"stream\":{\"version\":\"_0_0_1\",\"metadata\":{\"version\":\"_0_1_0\",\"language\":\"en\",\"title\":\"test upload\",\"description\":\"edit metadata, remove channel, add 1 lbc cost\",\"author\":\"\",\"license\":\"None\",\"nsfw\":false,\"thumbnail\":\"https://spee.ch/b/he0GRirDLvNMCl9PCAj52MWY.jpg\",\"preview\":\"\",\"licenseUrl\":\"\"},\"source\":{\"version\":\"_0_0_1\",\"sourceType\":\"lbry_sd_hash\",\"source\":\"LphfOfseLZfxZYN58mXh4CSmp0Jg8NPwKLEnIgg2SWUqYVUPjG1bbyO567BaOROU\",\"contentType\":\"image/jpeg\"}},\"publisherSignature\":{\"version\":\"_0_0_1\",\"signatureType\":\"SECP256k1\",\"signature\":\"ikct3ZF8Y4WS8BjHzyNoZQuh1GRoYTvXFgxYKIwg9xfeXo3nvPEQ8Jt6OW1U1wQ1IdztxSbrwp+lHX5W6PDBNw==\",\"certificateId\":\"RhsyLENHq/Daqmwwo4TseR8Ldz0=\"}}}",
      "version": "_0_0_1",
      "vout": 0
    }
  ]
}```

**Comment by tiger5226** [comment]
solved with 1.5.0 release
